src/Stack/Types/BuildPlan.hs

```diff
@@ -137,9 +137,9 @@ configuration. Otherwise, we don't cache.
 
 loadedSnapshotVC :: VersionConfig LoadedSnapshot
 #if MIN_VERSION_template_haskell(2,14,0)
-loadedSnapshotVC = storeVersionConfig "ls-v6" "RV5MidVhkF2v-PlFB5-tfpcX5xQ="
+loadedSnapshotVC = storeVersionConfig "ls-v7" "LVOy5Emee44pWXVQYpWbT6TaNy4="
```

Comment:
This is perfect, thanks. This code will actually be going away pretty soon with some in-flight changes, but this is absolutely the right way to handle it today.

Reply:
Thanks. It seems to be gone now ;)